.github/workflows/validate-linux-binaries.yml

```diff
@@ -36,8 +39,11 @@ jobs:
       fail-fast: false
     runs-on: ${{ matrix.validation_runner }}
     steps:
-      - name: Checkout PyTorch builder
+      - name: Checkout PyTorch atalman/builder
         uses: actions/checkout@v2
+        with:
+          ref: main
+          repository: atalman/builder
```

Comment:
This is temp change, otherwise the checkout/builder would use the code from builder/main that does not have the changes to the test

Reply:
> This is temp change, otherwise the checkout/builder would use the code from builder/main that does not have the changes to the test

Either this workflow is written in a horribly broken way, or above statement is wrong; checkout action conventionally works for pull requests without any changes